src/toil/test/src/regularLogTest.py

```diff
@@ -91,5 +91,5 @@ def testRegularLog(self):
                                               './toilTest',
                                               '--clean=always',
                                               '--batchSystem=singleMachine',
-                                              '--logLevel=info'], stderr=subprocess.STDOUT)
+                                              '--logLevel=debug'], stderr=subprocess.STDOUT)
```

Comment:
I think the log level option here should be deleted, just use the default.

Reply:
I'd keep this as info, just to reduce logging.

Reply:
Yes!